ember-cli-build.js

```diff
@@ -300,7 +300,15 @@ module.exports = function(options) {
   });
 
   emberTestingBundle = concat(emberTestingBundle, {
-    outputFile: 'ember-testing.js'
+    outputFile: 'ember-testing.js',
+    hasBootstrap: false,
+    footer: stripIndent`
+      var testing = requireModule('ember-testing');
+      Ember.Test = testing.Test;
```

Comment:
Confirmed that this works as intended via shared asset in https://github.com/emberjs/ember.js/pull/15112#issuecomment-292230312